Files/Views/ColumnShellPage.xaml

```diff
@@ -195,6 +195,7 @@
             HorizontalAlignment="Stretch"
             x:FieldModifier="public"
             Background="{ThemeResource SolidBackgroundFillColorBaseBrush}"
+            CacheSize="1"
```

Comment:
Is this still needed?

Reply:
Yeah this has been added in this PR. It looks like it has some benefit in terms of memory usage.

Reply:
@gave92 Actually, I wasn't sure if we should keep them or not. It's really up to you, and I wasn't sure how great the benefit is, if any

Reply:
Looking at VS memory profiler it looks like this makes memory occupation worse lol. Let me take another look on this.

Reply:
Yeah it still looks like we have a memory leak problem, the old layout is not getting collected when switching layoutmode and adding `CacheSize="1"` make the issue worse since a new layout is created each time layoutmode is changed.